engines/bops_preapps/app/views/bops_preapps/validation_requests/show/_other_change.html.erb

```diff
@@ -0,0 +1,64 @@
+<% content_for :page_title do %>
+  Other validation request &mdash; <%= t("page_title") %>
+<% end %>
+
+<h1>
+  <% if @validation_request.closed? %>
+    Check the response to other request
+  <% else %>
+    View other request
+  <% end %>
+</h1>
+<%= render "shared/proposal_header" %>
+
+<div class="govuk-grid-row govuk-!-margin-top-3">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-!-margin-bottom-3">Officer request</h2>
+
+    <div class="govuk-inset-text govuk-!-margin-top-3">
+      <h3 class="govuk-!-margin-bottom-1">Reason it is invalid:</h3>
+      <%= render(FormattedContentComponent.new(text: @validation_request.reason)) %>
+
+      <h3 class="govuk-!-margin-bottom-1">How it can be made valid:</h3>
+      <%= render(FormattedContentComponent.new(text: @validation_request.suggestion)) %>
+
+      <p>
+        <%= @validation_request.created_at.to_fs %>
+      </p>
+    </div>
+
+    <h2 class="govuk-!-margin-bottom-3">Applicant response</h2>
+
+    <div class="govuk-inset-text govuk-!-margin-top-3">
+      <% if @validation_request.closed? %>
+        <%= render(FormattedContentComponent.new(text: @validation_request.response)) %>
+
+        <p>
+          <%= @validation_request.updated_at.to_fs %>
+        </p>
+      <% else %>
+        <p>Applicant has not responded yet</p>
+      <% end %>
+    </div>
```

Comment:
Currently this block is hidden unless the planning application is in an invalidated state - seems like you'd want to see the applicant response even after the application has been validated so I've removed the condition.